assets/src/edit-story/app/font/fontProvider.js

```diff
@@ -28,11 +28,26 @@ import Context from './context';
 import useLoadFonts from './effects/useLoadFonts';
 import useLoadFontFiles from './actions/useLoadFontFiles';
 
+const GOOGLE_MENU_FONT_URL = 'https://fonts.googleapis.com/css';
+
 function FontProvider({ children }) {
+  const loadedFontFamily = useRef([]);
```

Comment:
Why does this use `useRef` over `useState`?

Reply:
loadedFontFamily is just a variable to store data and don't need rerender while updating it.